godot_project/project_workspace/workspace_context/structure_context/selection_db/selection_db.gd

```diff
@@ -459,55 +459,5 @@ func apply_selection_snapshot(in_snapshot: Dictionary) -> void:
 	var atom_snapshot: Array = in_snapshot.atom_snapshot
 	var spring_snapshot: Array = in_snapshot.spring_snapshot
 	
-	var atoms_to_selection_status: Dictionary = {
-		# atom_id<int> : is_atom_currently_selected<bool>
-	}
-	
-	var bond_to_selection_status: Dictionary = {
-		# bond_id<int> : is_bond_currently_selected<bool>
-	}
-	var spring_to_selection_status: Dictionary = {
-		# spring_id<int> : is_spring_selected<bool>
-	}
-	
```

Comment:
This is dead code that should have been removed when merging the snapshot system.

It was there when we had to manually call `representation.highlight_atoms()` after changing a selection, but that's not the case anymore.